src/Factory/OrderFactory.php

```diff
@@ -73,4 +76,20 @@ public function createForCustomer(string $customerEmail): OrderInterface
 
         return $order;
     }
+
+    private function getNewOrderCustomer(string $email): CustomerInterface
```

Comment:
```getNewOrderCustomer()``` -> ```getNewCustomerForOrder()``` ?

What I mean is that for me the method name is a little bit misleading - is the customer specified as the one who was created during admin order creation?

Reply:
👍 Maybe I would vote for `getCustomerForOrder`, wdyt? As it can be either existing or new customer

Reply:
Agreed, the best name of those mentioned above!